src/js/serial.js

```diff
@@ -292,71 +241,68 @@ var serial = {
         });
     },
     getInfo: function (callback) {
-        var chromeType = (this.connectionType == 'serial') ? chrome.serial : chrome.sockets.tcp;
+        const chromeType = (this.connectionType === 'serial') ? chrome.serial : chrome.sockets.tcp;
```

Comment:
`connectionType`?

Reply:
Set to `tcp` or `serial` for using the right chrome API. (Not being used in `getInfo` thus removed)